raiden/network/transport/matrix.py

```diff
@@ -698,7 +806,7 @@ def _get_unlisted_room(self, room_name, invitees):
                 else:
                     self.log.info('Room joined successfully', room=room)
                     return room
-            gevent.sleep(self._login_retry_wait)
+            gevent.sleep(self._config.get('login_retry_wait', 0.5))
```

Comment:
`0.5` should be a constant define in `settings.py`

Reply:
I just moved this code around, but ok